crates/aptos-crypto/src/hash.rs

```diff
@@ -144,6 +144,11 @@ impl HashValue {
             .map(Self::new)
     }
 
+    /// Get byte array representation.
+    pub fn get_bytes(&self) -> [u8; HashValue::LENGTH] {
```

Comment:
If you are cloning it, you might as well just use `HashValue::to_vec()` instead of defining a new method.

Reply:
See https://github.com/aptos-labs/aptos-core/blob/92f8b659094f2643a33a0d98f74b41dd676144f5/crates/aptos-crypto/src/hash.rs#L148